react/components/SearchResult.js

```diff
@@ -152,6 +153,8 @@ class SearchResult extends Component {
     const hideFacets = !map || !map.length
     const showLoading = loading && !fetchMoreLoading
     const showContentLoader = showLoading && !showLoadingAsOverlay
+    const getLastName = compose(prop('name'), last)
```

Comment:
Make this a method or an outside function so you don't recreate function on every render.